quoting-service/templates/config.yaml

```diff
@@ -9,4 +9,6 @@ metadata:
     app.kubernetes.io/managed-by: {{ .Release.Service }}
     helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
 data:
-  default.json: {{ (tpl (.Files.Get "configs/default.json") . ) | quote }}
\ No newline at end of file
+  default.json: {{ (tpl (.Files.Get "configs/default.json") . ) | quote }}
+  rules.json: {{ .Values.rules | quote }}
```

Comment:
I think it would be cleaner if you followed this approach: `rules.json: {{ .Values.rules  | toPrettyJson | quote }}`